SCAutolib/controller.py

```diff
@@ -429,9 +596,16 @@ def enroll_card(self, card: card.VirtualCard):
 
     def cleanup(self):
         """
-        Clean the system after setup. This method restores the SSSD config file,
-        deletes created users with cards, remove CA's (local and/or IPA Client)
+        Cleans up all system configurations and changes made by SCAutolib's
+        ``prepare`` command. This includes restoring SSSD
+        configuration, deleting created users (except 'root'), removing smart
+        cards and their associated directories, and cleaning up both local and
+        IPA client CA setups. It also clears
+        OpenSC and SSSD caches.
```

Comment:
nit: fits well on one line ;)
```suggestion
        IPA client CA setups. It also clears OpenSC and SSSD caches.
```